notification-portlet-webapp/src/main/webapp/scripts/modal-notice.js

```diff
@@ -0,0 +1,151 @@
+/*
+ * Licensed to Apereo under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work
+ * for additional information regarding copyright ownership.
+ * Apereo licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file
+ * except in compliance with the License.  You may obtain a
+ * copy of the License at the following location:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+/*
+ * This file leverages Modal Bootstrap to display a single notice, like an Term of Service
+ */
+
+var upmodal_notice = upmodal_notice || {};
+
+if (!upmodal_notice.init) {
+
+  upmodal_notice.init = true;
+
+  (function() {
+
+    var defaults = {
+      selectors: {
+        content:         '.modal-content',
+        title:           '.title',
+        body:            '.body',
+        link:            '.link',
+        actions:         '.notification-actions',
+        actionTemplate:  '.action-template'
+      },
+      readyCallback: function() {}
+    };
+
+    // First 'prime-the-pump' with an ActionURL
+    function initNotices($, settings, callback) {
+      $.ajax({
+        type: 'POST',
```

Comment:
:art: avoid mixed single space and horizontally aligned object values. (lines 33-38 vs 46-51)